packages/coinstac-ui/app/render/state/apollo-client.js

```diff
@@ -44,5 +46,16 @@ export default function getApolloClient(config) {
     },
   }]);
 
+  client.networkInterface.useAfter([{
+    applyBatchAfterware(res, next) {
+      const statusText = get(res, 'responses.0.statusText');
+      if (statusText === 'Unauthorized') {
+        ipcRenderer.send('token-expired', { message: 'Expired token' });
```

Comment:
Same constant talk here. Either pass an object with a code property with a constant like *EXPIRED_TOKEN* or pass the message you want from here...